addon/components/bs-contextual-help/element.js

```diff
@@ -113,16 +98,15 @@ export default class ContextualHelpElement extends Component {
    * @type {object}
    * @private
    */
-  @computed('arrowClass', 'autoPlacement', 'viewportElement', 'viewportPadding')
   get popperModifiers() {
-    let self = this;
+    let id = this.args.id;
```

Comment:
```suggestion
    let { id } = this.args;
```

Just coding style. Feel free to ignore.